target/linux/ramips/mt7620/base-files/etc/board.d/01_leds

```diff
@@ -129,6 +129,9 @@ hnet,c108)
 	ucidef_set_led_netdev "lan" "lan" "$boardname:green:lan" "eth0"
 	ucidef_set_led_netdev "modem" "modem" "$boardname:green:modem" "wwan0"
 	;;
+hootoo,tm05)
+        set_wifi_led "$boardname:green"
```

Comment:
Please use trigger in DTS.

Reply:
Rather than in 01_leds you mean? Sure, I can move that over. Do you know the syntax? Thanks!

Reply:
https://github.com/openwrt/openwrt/blob/master/target/linux/ramips/dts/mt7620a_edimax_ew-7478ac.dts#L33

Try phy0tpt first and if that does not work, try phy0radio. First will flash on activity, second will only show whether WiFi is on/off.

Reply:
OK, trigger added to DTS, removed this in 01_leds. Will let you know. Thanks!

Reply:
This is still here?

Reply:
Removed - I missed that one, sorry. Thanks for catching it!